tests/tls/ca.crt

```diff
@@ -0,0 +1,29 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
```Validity
            Not Before: Jul  2 22:26:18 2024 GMT
            Not After : Aug 31 22:26:18 2024 GMT
```

Can we either make this expire quite far out in the future or generate the cert ephemerally on each test run?